clients/cobol-lsp-vscode-extension/src/services/Middleware.ts

```diff
@@ -0,0 +1,42 @@
+import { CancellationToken } from "vscode";
+import { ConfigurationParams, ConfigurationRequest } from "vscode-languageclient";
+import { CopybooksDownloader } from "./CopybooksDownloader";
+import { CopybooksPathGenerator } from "./CopybooksPathGenerator";
+
+export class Middleware {
```

Comment:
Middleware is a function. Consider export just function instead of class.

Reply:
I see, from TS doc is recommended to use export of functions to avoid to bring the class namespace that add pollution into the import section (and probably at compile time)

Reply:
It's still possible to use namespace instead of class.

Reply:
👍

Reply:
I need to export the class because I make an instance of the Middleware in `extention.ts`.
Inside `handleConfigurationRequest` I use copybooksPathGenerator and copybookDownloader, and I don't know how to pass them otherwise than in constructor.